xclim/indicators/atmos/_precip.py

```diff
@@ -166,6 +166,7 @@ class PrTas(Daily2D):
 fire_weather_indexes = Daily(
     _nvar=4,
     identifier="FWI",
+    realm="atmos",
```

Comment:
Why does this need to be defined ?

Reply:
Because the module of a class is the file in which the class is defined, not where it is instantiated. Meaning that as we are using "xclim.core.indicator.Daily" and not one of the classes defined under `xclim.indicators.atmos", the class's `__new__` doesn't "detect" the realm from the module... 
Maybe a more appropriate technique would be to create a "fire weather indexes" class that would have only one instance?